bertinNDAYISHIMIYE/capstoneProject-MyBrand
# ft-delete-enquiry

###What does this PR do?

>add delete enquiry route
>add delete enquiry logic into controllers

###Description of the task to be completed!

>Check if the user is authorised
>Allow authorised user to delete an enquiry

###How should this be tested manually?

>Clone this repo with: https://github.com/bertinNDAYISHIMIYE/capstoneProject-MyBrand
>Checkout to ft-deleteEnquiry branch
>Make sure node.js is installed
>Run command npm init for npm initialisation
>Run npm i to install different packages
>Use postman as Third party software to test the end point

###Any background context you want to provide?
>N/P
###Relavant trello card
https://trello.com/c/ue7kFlkY

###Screenshot

<img width="1419" alt="Screen Shot 2020-12-09 at 19 23 45" src="https://user-images.githubusercontent.com/72761713/101665627-e398b880-3a55-11eb-8412-0d676f9f36e8.png">
